submissions/BrianneHostutler.md

```diff
@@ -0,0 +1,10 @@
+* submitter: @BrianneHostutler
+* statement: I hereby agree that the facts I have submitted are my own, and that I grant a license for Pryaxis Limited, or designated entities by Pryaxis Limited, the ability to use this data in a way to promote future Pryaxis services.
+
+----
+
+* The Havanese is a dog breed of Bichon type.
```

Comment:
References to @[Havanese] need to have @ in front of them, otherwise great submissions thus far!